recipes/libint2/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -x
+echo ${PREFIX}
+export CXXFLAGS="${CXXFLAGS} -O2 -pipe -march=x86-64 -std=c++11 -fPIC -fopenmp"
```

Comment:
Do you need any of these flags other than `-fopenmp`? Many of them are already set in conda's default value of `CXXFLAGS`, from the CircleCI log:
```bash
+CXXFLAGS=-fvisibility-inlines-hidden -std=c++17 -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -pipe -I$PREFIX/include -fdebug-prefix-map=${SRC_DIR}=/usr/local/src/conda/${PKG_NAME}-${PKG_VERSION} -fdebug-prefix-map=${PREFIX}=/usr/local/src/conda-prefix
```

Reply:
I only need `fPIC` and `fopenmp`

Reply:
`-fPIC` is already included in conda's default flags

Reply:
done